pkg/sypgp/sypgp.go

```diff
@@ -258,6 +258,22 @@ func LoadPrivKeyring() (openpgp.EntityList, error) {
 	return el, nil
 }
 
+// LoadPubKeyringFromFile loads the public key from a local file given from a path into an EntityList
+func LoadPubKeyringFromFile(path string) (openpgp.EntityList, error) {
+
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+
+	el, err := openpgp.ReadKeyRing(f)
```

Comment:
note that because of the structure of this code, you can change this to:

    return openpgp.ReadKeyRing(f)